pkg/ui/src/views/shared/components/dropdown/index.tsx

```diff
@@ -71,12 +71,7 @@ export default class Dropdown extends React.Component<DropdownOwnProps, {}> {
     }
   }
 
-  arrowRenderer = ({ isOpen }: { isOpen: boolean }) => {
-    if (!isOpen) {
-      return <span><Icon type="caret-up" /></span>;
-    }
-    return <span className="active"><Icon type="caret-down" /></span>;
-  }
+  arrowRenderer = () => <span className="active"><Icon type="caret-up" /></span>;
```

Comment:
Does this only ever render an "up caret" now?

Reply:
yes, that was an actual request. but after it will be redesigned again. see https://github.com/cockroachdb/cockroach/pull/45802

Reply:
👍